assets/js/main.js

```diff
@@ -0,0 +1,29 @@
+const navbtn = document.querySelector(".nav-btn");
+const sidemenu = document.querySelector(".sidemenu");
+const closebtn = document.querySelector(".close-btn");
+
+navbtn.addEventListener("click", () => {
+    navbtn.classList.toggle("active");
+    sidemenu.classList.toggle("active");
+})
```

Comment:
Codacy has a fix for the issue: [Insert `;`](https://app.codacy.com/gh/aerial978/blog/pullRequest?prid=9584605)
```suggestion
});
```